mbf_abstract_nav/src/controller_action.cpp

```diff
@@ -310,6 +310,13 @@ void ControllerAction::runImpl(GoalHandle &goal_handle, AbstractControllerExecut
         goal_handle.setAborted(result, result.message);
         break;
 
+      case AbstractControllerExecution::ROBOT_DISABLED:
+        ROS_FATAL_STREAM_NAMED(name_, "Robot stopped and ignored velocity commands!");
```

Comment:
```suggestion
        ROS_FATAL_STREAM_NAMED(name_, "Robot is stopped and ignoring velocity commands!");
```
S: phrasing (same comment for the next line below)

Reply:
I changed it to "Robot ignored velocity commands for more than tolerance time". It makes more sense to me to write it in past tense since it already happened for N seconds.